### ISSUE TITLE
Docs: NodeJS sdk doc is blank on clicking from SDK overview page

### ISSUE THREAD

**Issue by ynot3363** [Infisical/infisical#2560]
Path: /sdks/languages/node

Blank page that should point to this repo: https://github.com/Infisical/node-sdk-v2. Linked from https://infisical.com/docs/documentation/getting-started/sdks

**Comment by ynot3363** [comment]
Yeah but that is a separate link, why even bring users to that blank page instead of the repo itself?

**Comment by kasyap1234** [comment]
yeah that's a valid point , maybe they are planning to add something there (on the blank page )

**Comment by akhilmhdh** [comment]
Closing as resolved
